### ISSUE TITLE
Not working for gems with a gemspec but no Gemfile

### ISSUE THREAD

**Issue by troessner** [sirlantis/rubocop-for-rubymine#26]
I did check existing and past issues but nobody seems to have reported this already.

I use rubymine for both [Reek](https://github.com/troessner/reek) and [RubyCritic](https://github.com/whitesmith/rubycritic). Both projects use rubocop but since they are gems, rubocop is listed in the gemspec and not the Gemfile.

Is this something you have on your roadmap?

**Comment by sirlantis** [comment]
The `Gemfile` is only used to decide whether the command should be prefixed with `bundle exec`.

How dou you invoke `rubocop` from the CLI?

**Comment by sirlantis** [comment]
Also, out of curiosity: does it work if you put in a `Gemfile` with

```
source "http://www.rubygems.org"
gemspec
```

and run `bundle install`?

**Comment by troessner** [comment]
>> Also, out of curiosity: does it work if you put in a Gemfile with
source "http://www.rubygems.org"
gemspec

No, it is [already like that](https://github.com/troessner/reek/blob/master/Gemfile#L4).

Please see the screenshot attached as well.
<img width="482" alt="screen shot 2015-11-24 at 11 33 06" src="https://cloud.githubusercontent.com/assets/271109/11393303/3c0dd7a0-935f-11e5-9507-33a76fb87334.png">

**Comment by sirlantis** [comment]
So, I see this happening in the logs:

```
2015-11-25 15:29:11,246 [ 211606]  DEBUG - hub.sirlantis.rubymine.rubocop - Executing RuboCop (SDK=/usr/local/var/rbenv/versions/2.0.0-p647/bin, Bundler=true)/usr/local/var/rbenv/versions/2.0.0-p647/bin/ruby rubocop --format json /Users/marceljackwerth/Workspace/github/reek/lib/reek.rb 
```

It thinks it should use bundler (which is correct) but then it doesn't use `bundle exec` in the call.

Wondering if this is a regression from #9 / https://github.com/sirlantis/rubocop-for-rubymine/commit/b755a0efcce40b1ce4bba7e82096d05b4d9f02fc

**Comment by cayblood** [comment]
@sirlantis I'm seeing this same error after installing the plugin in rubymine and running inside a project with a Gemfile and the latest version of rubocop. Any troubleshooting/logging you'd like me to run?

**Comment by sirlantis** [comment]
I reproduced the issue and it should be fixed by 3.0.1 (submitted to Jetbrains right now)
